k8s-bench/main.go

```diff
@@ -30,14 +30,19 @@ import (
 )
 
 type Task struct {
-	Goal       string `json:"goal"`
 	Setup      string `json:"setup,omitempty"`
 	Verifier   string `json:"verifier,omitempty"`
 	Cleanup    string `json:"cleanup,omitempty"`
 	Difficulty string `json:"difficulty"`
 	Disabled   bool   `json:"disabled,omitempty"`
 
 	Expect []Expectation `json:"expect,omitempty"`
+
+	Script []ScriptStep `json:"script,omitempty"`
```

Comment:
Not sure what's happening here. Does `Script` here means `ChatMessages` or `ChatScript` ?

Reply:
Script is the prompts we send.  It replaces `goal`, and allows us to have multiple prompts